### ISSUE TITLE
TVDB Library Info in Alpha 10

### ISSUE THREAD

**Issue by thomfam** [koying/SPMC#1029]
Hi Koying, sorry but the problem with TVDB still exists with Alpha 10.  Today I went to add the series Mr Mercedes to my Library,  I got the overall information OK but not the information for each episode after updating to Alpha 10.  Went to my 17.4 version of Krypton, files, highlighted Mr Mercedes folder, then Context Menu, Information and refresh and info for the current 4 episodes was added fine.  Reverted to Alpha 8 in SPMC, highlighted folder, the Context Menu, Information and refresh and this time the episode information for the first 4 episodes was added.

**Comment by Michelasso** [comment]
It seems to work fine with me. These are the steps I followed for testing. I don't have a new TV series, so I had to manage with the old ones:

- SMPC 17.4.a8 was installed and all TV Shows were showing fine
- Upgraded to 17.4.a10. All TV Shows were there
- With the file manager in the server I moved "The Walking Dead" folder, with Season 7 inside, out of my TV Shows folder
- Refreshed. "The Walking Dead" disappeared
- Moved "The Walking Dead" back to TV Shows folder. It does a partial refresh. The show had its icon, but the episodes count was 0/0
- Clicked on the series, got info and refreshed. All episodes appeared

**Comment by Kr1ss-XD** [comment]
@thomfam I recognized as well that new series won't be scraped with alpha-10. Thank you for highlighting this again.
My workaround for the time being is to scrape the series with an older version (17.4a8 or Jarvis), export to separate 
.nfo files and import them in alpha-10.
Btw this issue seems to be independent from the scraper, because I also tried the TMDB scraper for shows, getting the same result (series is scraped but no episodes).

**Comment by Michelasso** [comment]
Ok, I confirm that adding a new series it doesn't work. I added "The Originals/The Originals Series 4" and in the main page there is nothing. If i go into the Tag "Title" it shows, but the icon says 0/0 episodes.

I forgot in my previous test that the info of the series I removed was probably still cached.

**Comment by mLgz0rn** [comment]
I'm also experiencing this issue, both in alpha 9 and 10.
It detects the tv shows, but not the seasons or episodes.
It's also marking all my tv shows as watched, but guess that could be because it's not detecting any episodes.
I reverted back to alpha 8, and it works fine there.

**Comment by Michelasso** [comment]
Yes, same here. I downgraded to alpha 8 as well now. I could manage to add the new series only removing and adding the shared folder again, though. Just refreshing the library didn't do anything.

**Comment by Kelso76** [comment]
Hi everyone! Does downgrading to alpha8 wipe all spmc data? Is it like an update or a separate install ? Thanks

**Comment by Kr1ss-XD** [comment]
@Kelso76 All data will be kept when downgrading from a10 to a8

**Comment by Kelso76** [comment]
Thank you !

**Comment by Biibou** [comment]
Same issue here with 17.4a10 and 17.4a9. Everything is OK with 17.4a8.
Here is a log file with alpha 8 and alpha 9 (both fresh install)
https://www.dropbox.com/s/7q4qgtfju5fupl6/17.4a9.log?dl=0
https://www.dropbox.com/s/7tq8jxzzh04xujd/17.4a8.log?dl=0

With 17.4a8 there is :
DEBUG: GetEpisodeList: Searching 'http://thetvdb.com/api/439DFEBA9D3059C6/series/323168/all/fr.zip' using The TVDB scraper (file: '/storage/emulated/0/Android/data/com.semperpax.spmc17/files/.spmc/addons/metadata.tvdb.com', content: 'tvshows', version: '1.8.5')

With 17.4a9 this line is missing and :
ERROR: VideoInfoScanner: Asked to lookup episode MYVIDEOFILE.mkv online, but we have no episode guide. Check your tvshow.nfo and make sure the <episodeguide> tag is in place.

**Comment by spec1re** [comment]
Thanks! Downgrading to a8 did the trick and browsing thru the library is much smother here on the Mi Box 4K as with a10, someone know why? With a8, I noticed Sysinfo > Info shows all CPU cores, with a10 only 3 are visible, the 4th is cut off.

**Comment by Michelasso** [comment]
I couldn't test it completely (I have not a new TV show to add), but adding single episodes in an already present show  they get fully recognised in alpha 11. Is it fixed for good, now?
